crates/notion-core/src/toolchain/mod.rs

```diff
@@ -123,16 +128,15 @@ impl Toolchain {
 
     pub fn save(&self) -> Fallible<()> {
         let path = user_platform_file()?;
-        let mut file = File::create(&path).unknown()?;
-        match &self.platform {
-            &Some(ref platform) => {
+        let result = match &self.platform {
+            Some(ref platform) => {
```

Comment:
We can drop `ref` here, courtesy of 2018 Edition.

Reply:
Excellent! Is that true in all pattern matching?

Reply:
*Many*. The cases where it can't, the compiler will tell you.